src/array.ts

```diff
@@ -1,5 +1,6 @@
-import { MAX_SAFE_INTEGER as maxSafeInteger } from './number';
 import has from './has';
+import { forOf, isArrayLike, isIterable, Iterable } from './Iterator';
+import { MAX_SAFE_INTEGER as maxSafeInteger } from './number';
 
 export interface ArrayLike<T> {
```

Comment:
Is `interface ArrayLike<T>` needed, since it's provided by TypeScript in lib.core.d.ts?